src/Command/Processor/KeyPrefixProcessor.php

```diff
@@ -406,6 +409,23 @@ public static function evalKeys(CommandInterface $command, $prefix)
         }
     }
 
+    /**
+     * Applies the specified prefix to the keys of an FCall-based command.
+     *
+     * @param CommandInterface $command Command instance.
+     * @param string           $prefix  Prefix string.
+     */
+    public static function fCallKeys(CommandInterface $command, $prefix)
```

Comment:
@mehrud Need to add a check for keys argument. In `FCALL` interface only `$function` argument is required, so if no keys or arguments will be provided it will throw `Undefined idnex: $argument[1]`

Reply:
@vladvildanov It's OK, since in FCALL command, like in EVAL, function name is always the first argument, and key numbers is the second one, so if there's no argument provided by the user, the `$command->getArguments()` will result in ['function_name', 0]

Reply:
@mehrud Right, I missed that this one executes after `setArguments()`